Broken Windows CI: "Cannot find path 'D:\a\servo\servo\resources' because it does not exist"

https://github.com/servo/servo/actions/runs/15499890835/job/43645261161
https://github.com/servo/servo/actions/runs/15499278941/job/43643352149
https://github.com/servo/servo/actions/runs/15497749370/job/43638622873
https://github.com/servo/servo/actions/runs/15496984941/job/43636178946

```

Run cp D:\a\servo\servo\resources C:\a\servo\servo -Recurse
Copy-Item: C:\a\_temp\45a77269-da55-4211-bf6a-d6953ba932c2.ps1:2
Line |
   2 |  cp D:\a\servo\servo\resources C:\a\servo\servo -Recurse
     |  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot find path 'D:\a\servo\servo\resources' because it does not exist.
Error: Process completed with exit code 1.
```

--- a/.github/workflows/windows.yml
+++ b/.github/workflows/windows.yml
@@ -167,10 +167,6 @@ jobs:
         run: |
           .\mach build --use-crown --locked --${{ inputs.profile }} ${{ inputs.build-args }}
           cp C:\a\servo\servo\target\cargo-timings C:\a\servo\servo\target\cargo-timings-windows -Recurse
-      - name: Copy resources
-        if: ${{ runner.environment != 'self-hosted' }}
-        # GitHub-hosted runners check out the repo on D: drive.
-        run: cp D:\a\servo\servo\resources C:\a\servo\servo -Recurse
       - name: Smoketest
         run: .\mach smoketest --${{ inputs.profile }}
       - name: Unit tests

